pkg/listening-sockets/listening-sockets.go

```diff
@@ -211,26 +213,28 @@ func (cc *ConnectionCheck) extractContainerID(debugPod *corev1.Pod, pid string)
 	return containerID, nil
 }
 
-// extractContainerName receives CRI-O container ID and returns the container's name.
-func (cc *ConnectionCheck) extractContainerName(debugPod *corev1.Pod, containerID string) (string, error) {
+// extractContainerName receives CRI-O container ID and returns the container's Info(containerName, containerNameSpace, containerPod).
+func (cc *ConnectionCheck) extractContainerInfo(debugPod *corev1.Pod, containerID string) (string, string, string, error) {
```

Comment:
Maybe this func can return a container obj instead of returning a list of vars that might get bigger in the future if we decide to add additional container info